Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM centos:7
+ENV container docker
+
+# Set locale manually on build (docker defaults to POSIX which causes initdb to set the wrong db encoding, MIQ must have UTF8)
+# Once systemd is online, it will set locale based /etc/locale.conf
+
+# Only needed if building with docker-1.8
+ENV LANG en_US.UTF-8  
+
+# Install EPEL repo, yum necessary packages for the build without docs, clean all caches (no DNF)
+RUN rpm -ivh https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm && yum -y install --setopt=tsflags=nodocs nodejs tar sudo git memcached postgresql-devel postgresql-server libxml2-devel libxslt-devel patch gcc-c++ openssl-devel gnupg curl which net-tools libyaml-devel autoconf readline-devel libffi-devel bzip2 automake libtool bison sqlite-devel ; yum clean all
+
+# Systemd cleanups, setup volumes
+RUN (cd /lib/systemd/system/sysinit.target.wants/; for i in *; do [ $i == systemd-tmpfiles-setup.service ] || rm -f $i; done); \
+rm -f /lib/systemd/system/multi-user.target.wants/*;\
+rm -f /etc/systemd/system/*.wants/*;\
+rm -f /lib/systemd/system/local-fs.target.wants/*; \
+rm -f /lib/systemd/system/sockets.target.wants/*udev*; \
+rm -f /lib/systemd/system/sockets.target.wants/*initctl*; \
+rm -f /lib/systemd/system/basic.target.wants/*;\
+rm -f /lib/systemd/system/anaconda.target.wants/*;
+VOLUME [ "/sys/fs/cgroup" ]
+
+# POSTGRESQL INIT
+# Do not call a login "-" su, resets the ENV
+RUN su postgres -c 'initdb -D /var/lib/pgsql/data'
+
+## 2. RVM
+# Download and install RVM, setup environment, install ruby/gems, clean all
+# Note: RVM uses yum to bring missing pre-reqs
+
+RUN /usr/bin/curl -sSL https://rvm.io/mpapis.asc | gpg2 --import - && /usr/bin/curl -sSL https://get.rvm.io | rvm_tar_command=tar bash -s stable && source /etc/profile.d/rvm.sh && echo "gem: --no-ri --no-rdoc --no-document" > ~/.gemrc && /bin/bash -l -c "rvm requirements ; rvm install ruby 2.2.3 ; rvm use 2.2.3 --default ; gem install bundler rake ; gem install nokogiri -- --use-system-libraries ; rvm cleanup all ; yum clean all ; rvm disk-usage all"
+
+# GIT clone and prepare services (shallow clone)
+RUN mkdir /manageiq && git clone --depth 1 https://github.com/ManageIQ/manageiq /manageiq
```

Comment:
@bazulay Yes, we leverage the bybusiness load balancer feature of httpd to split the load across our multiple UI workers.  We have code that currently depends heavily on httpd files and service.